### DISCUSSION TITLE
Update 2020-11-16

### DISCUSSION THREAD

**Discussion by frol** [near/near-explorer#481] [Show and tell]
Our previous update is here: #470

Find our Objectives and Key Results for 2020 Q4: #467

# Summary by Focus

### Explorer for Non-tech Users

* We have updated the logo according to the new https://near.org/brand
* Explorer redesign is on its way (we even had a brief moment rolling it out to production, but decided to undo it to make final touches). @icerove and @corwinharrell are iterating on it #466 (PRs: #466, #483)

### Explorer for Contract Developers

* Indexer for Explorer received more touches from @khorolets, and we currently have the indexers deployed for testnet and mainnet completely synced to the tip of the network! Explorer frontend is getting transitioned from the old database in #477 and #478 by @icerove 

### Explorer for Validators

* Design of the Validators Dashboard is getting more iterations from @corwinharrell based on the received feedback from the community :tada: See more details in #370

**Comment by khorolets** [comment]
Addressed the idempotent issue for adding Accounts and AccessKeys (https://github.com/near/near-indexer-for-explorer/issues/33) 

I've updated the binary for testnet and mainnet and restarted both Indexers from genesis. Testnet one is already up to date with testnet and running in real time. Mainnet Indexer is working a bit slower we are going to investigate the cause. 

Testnet showed that we have some issue with missing data, particularly chunks we are missing for some reason. I am working on investigation here https://github.com/near/near-indexer-for-explorer/issues/48 

The plan for next week is to deal with missing data issue and rerun the Indexers once again to ensure we have all the data without misses. Add storing of State Changes https://github.com/near/near-indexer-for-explorer/issues/9, move ExecutionOutcomes into corresponding Chunks https://github.com/near/near-indexer-for-explorer/issues/31

**Comment by heycorwin** [comment]
- Continuing to make progress on the validator dashboard after receiving feedback on the initial design direction
- Revisiting dashboard designs to ensure that the mockups communicate clearly about various breakpoints and other smaller design details
- Hoping to do some QA this week in collaboration with @icerove so I can assist in ensuring the implementation is consistent with the new designs.
- Continuing to go over the existing explorer views to suggest small UI changes in order to remain consistent with the new dashboard design
